src/common/components/cards/result-section-title.tsx

```diff
@@ -11,13 +11,21 @@ export type ResultSectionTitleProps = {
     title: string;
     badgeCount: number;
     outcomeType: InstanceOutcomeType;
+    shouldAlertFailuresCount?: boolean;
 };
 
 export const ResultSectionTitle = NamedFC<ResultSectionTitleProps>('ResultSectionTitle', props => {
+    const failureTerm = props.badgeCount > 1 ? 'failures were' : 'failure was';
```

Comment:
In a case with 0 failures, will this announce "0 failures was..." ?

Reply:
Good catch, I was fighting with some weird behavior and totally forgot about 0. Will update presently

Reply:
updated.